### PR TITLE
Adds Unit tests

### PR DESCRIPTION
**Pull request by vrindle**
More unit tests need to be added.

### CODE REVIEW

File: library/firewall_lib.py [hunk 1]
```diff
@@ -170,11 +170,12 @@
 
     FW_VERSION = firewall.config.VERSION
 
-    from firewall.client import FirewallClient, Rich_Rule, FirewallClientZoneSettings
 
+    from firewall.client import FirewallClient, Rich_Rule, FirewallClientZoneSettings
     HAS_FIREWALLD = True
 except ImportError:
     HAS_FIREWALLD = False
+    FirewallClient=None
```
[the remark this answers is not in the document]
Comment (vrindle) [reply library/firewall_lib.py:178]: Hi Rich... I realized that I was making changes and the line above is necessary in order to properly mock out FirewallClient. Patching does not seem to work for some reason with conditional imports, if you know a better way to do this please let me know. Otherwise this is only I know to mock out FirewallClient.

File: library/firewall_lib.py [hunk 2]
```diff
@@ -175,6 +175,7 @@
     HAS_FIREWALLD = True
 except ImportError:
     HAS_FIREWALLD = False
+    FirewallClient = None
```
Comment (richm) [inline library/firewall_lib.py:178]: please revert this change

Comment (vrindle) [reply library/firewall_lib.py]: Reverted

**Review by richm — APPROVED** [review]
lgtm

**Comment by richm** [comment]
please change the PR title to remove the draft wording, then I think you can merge it
